zerver/webhooks/airbyte/fixtures/airbyte_job_payload_failure.json

```diff
@@ -0,0 +1,37 @@
+{
+    "data": {
+        "workspace": {
+            "id":"b510e39b-e9e2-4833-9a3a-963e51d35fb4",
+            "name":"Workspace1",
+            "url":"https://link/to/ws"
+        },
+        "connection":{
+            "id":"64d901a1-2520-4d91-93c8-9df438668ff0",
+            "name":"Connection",
+            "url":"https://link/to/connection"
+        },
+        "source":{
+            "id":"c0655b08-1511-4e72-b7da-24c5d54de532",
+            "name":"PostgreSQL",
+            "url":"https://link/to/source"
+        },
+        "destination":{
+            "id":"5621c38f-8048-4abb-85ca-b34ff8d9a298",
+            "name":"BigQuery",
+            "url":"https://link/to/destination"
+        },
+        "jobId":9988,
+        "startedAt":"2024-01-01T00:00:00Z",
+        "finishedAt":"2024-01-01T01:00:00Z",
+        "bytesEmitted":1000,
+        "bytesCommitted":1000,
+        "recordsEmitted":89,
+        "recordsCommitted":89,
+        "bytesEmittedFormatted": "1000 B",
+        "bytesCommittedFormatted":"90 B",
+        "success":false,
+        "durationInSeconds":3600,
+        "durationFormatted":"1 hours 0 min",
+        "errorMessage": "Connection timeout while trying to sync data"
+    }
+}
```

Comment:
Actual payload example:
```suggestion
{
  "text": "Your connection Connection from PostgreSQL to BigQuery just failed...\nThis happened with Checking source connection failed - please review this connection's configuration to prevent future syncs from failing\n\nYou can access its logs here: https://link/to/ws/connections/64d901a1-2520-4d91-93c8-9df438668ff0\n\nJob ID: 20405145",
  "blocks": [
    {
      "type": "section",
      "text": {
        "type": "mrkdwn",
        "text": "Sync failure occurred: <https://link/to/ws/connections/64d901a1-2520-4d91-93c8-9df438668ff0|Connection>"
      }
    },
    {
      "type": "section",
      "fields": [
        {
          "type": "mrkdwn",
          "text": "*Source:*"
        },
        {
          "type": "mrkdwn",
          "text": "<https://link/to/ws/source/c0655b08-1511-4e72-b7da-24c5d54de532|PostgreSQL>"
        },
        {
          "type": "mrkdwn",
          "text": "*Destination:*"
        },
        {
          "type": "mrkdwn",
          "text": "<https://link/to/ws/destination/5621c38f-8048-4abb-85ca-b34ff8d9a298|BigQuery>"
        },
        {
          "type": "mrkdwn",
          "text": "*Duration:*"
        },
        {
          "type": "mrkdwn",
          "text": "14 sec"
        }
      ]
    },
    {
      "type": "section",
      "text": {
        "type": "mrkdwn",
        "text": "*Failure reason:*\n\n```\nChecking source connection failed - please review this connection's configuration to prevent future syncs from failing\n```\n"
      }
    },
    {
      "type": "section",
      "text": {
        "type": "mrkdwn",
        "text": "*Sync Summary:*\n0 record(s) extracted / 0 record(s) loaded\n0 B extracted / 0 B loaded\n"
      }
    }
  ],
  "data": {
    "workspace": {
      "id": "b510e39b-e9e2-4833-9a3a-963e51d35fb4",
      "name": "Workspace1",
      "url": "https://link/to/ws"
    },
    "connection": {
      "id": "64d901a1-2520-4d91-93c8-9df438668ff0",
      "name": "Connection",
      "url": "https://link/to/ws/connections/64d901a1-2520-4d91-93c8-9df438668ff0"
    },
    "source": {
      "id": "c0655b08-1511-4e72-b7da-24c5d54de532",
      "name": "PostgreSQL",
      "url": "https://link/to/ws/source/c0655b08-1511-4e72-b7da-24c5d54de532"
    },
    "destination": {
      "id": "5621c38f-8048-4abb-85ca-b34ff8d9a298",
      "name": "BigQuery",
      "url": "https://link/to/ws/destination/5621c38f-8048-4abb-85ca-b34ff8d9a298"
    },
    "jobId": 20405145,
    "startedAt": "2024-10-22T07:13:48Z",
    "finishedAt": "2024-10-22T07:14:02Z",
    "bytesEmitted": 0,
    "bytesCommitted": 0,
    "recordsEmitted": 0,
    "recordsCommitted": 0,
    "errorMessage": "Checking source connection failed - please review this connection's configuration to prevent future syncs from failing",
    "durationFormatted": "14 sec",
    "bytesEmittedFormatted": "0 B",
    "bytesCommittedFormatted": "0 B",
    "success": false,
    "durationInSeconds": 14
  }
}
```